src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -555,6 +556,20 @@ public function load(array $configs, ContainerBuilder $container)
         $container->registerAttributeForAutoconfiguration(AsController::class, static function (ChildDefinition $definition, AsController $attribute): void {
             $definition->addTag('controller.service_arguments');
         });
+        $container->registerAttributeForAutoconfiguration(AsAlias::class, static function (ChildDefinition $definition, AsAlias $attribute, \ReflectionClass $class, string $serviceId, ContainerBuilder $container): void {
+            static $usedAttributeAliases = [];
```

Comment:
This should not be a static variable as this can lead to state sharing between instances (eg in tests)

Reply:
@nicolas-grekas Maybe I misunderstood you, but every call to `load` should create a new anonymous function with it's own static variable https://3v4l.org/iDth3 . I can move it to a property if that's more preferable.

Reply:
Oh, I missed this, thanks for the hint!